resources/hooks/useLogin.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { useNavigate } from "react-router";
+import { useUserStore } from "../js/useUserStore";
+import { apiUrl } from "../js/App";
+
+export const useLogin = () => {
+    const [errorMessage, setErrorMessage] = useState("");
```

Comment:
Je pense qu'on pourrait utiliser le hook : renderFieldError au lieu de ce state. Meme si tu n'as que 1 zone d'erreur sur le login, tu peux toujours faire un setErrors("login") et un renderFieldError là ou tu veux

Reply:
I intergated a system where we check the input by my validator rules (not empty, valid email address is required) and if there is any other error on the server, I send status=false and 'message' => __('auth.failed'). So this error message that was hardcoded is now in the lang/fr/auth.php. 
So I reused renderFieldError (useForm hook) for a general error and for the errors on the each input. 
Updated code in Login.jsx, useLogin and LoginController